datafusion/core/src/physical_plan/aggregates/mod.rs

```diff
@@ -72,6 +76,15 @@ pub enum AggregateMode {
     Single,
 }
 
+/// Group By expression modes
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub enum GroupByOrderMode {
+    /// Some of the expressions in the GROUP BY clause have an ordering.
```

Comment:
I wonder if it we could add a more precise definition of "partial" here

perhaps something like the "prefix of the group keys is some permutation of the sort key prefix "?

I think your inline comment below explains it nicely with an example:
```
    // Find out how many expressions of the existing ordering define ordering
    // for expressions in the GROUP BY clause. For example, if the input is
    // ordered by a, b, c, d and we group by b, a, d; the result below would be.
    // 2, meaning 2 elements (a, b) among the GROUP BY columns define ordering.
```

Maybe we can replicate that comment here in the documentation for GroupByOrderMode

Reply:
Fixed it